app/__main__.py

```diff
@@ -39,24 +39,25 @@
 from routes.ui.completed_page import UIScenarioCompletedPageHandler
 from routes.ui.landing_page import UIScenarioLandingPageHandler
 from routes.ui.scenario import UIScenarioHandler
-from routes.user.asset import UserAssetHandler
-from routes.user.loading_screen import UserLoadingScreen
-from routes.user.scenario import UserScenarioHandler
-from routes.user.scene import UserSceneHandler
-from routes.user.solved import UserSolvedHandler
+
+# from routes.user.asset import UserAssetHandler
+# from routes.user.loading_screen import UserLoadingScreen
+# from routes.user.scenario import UserScenarioHandler
+# from routes.user.scene import UserSceneHandler
+# from routes.user.solved import UserSolvedHandler
```

Comment:
Intentional commented out lines?

Reply:
Yup for both... these are the old user API routes that aren't used atm, so thought it's best to block them out of deployment. We can uncomment if we find out some are needed (for stats for example).

Reply:
I see. can also remove, but I'll leave that with you

Reply:
Yeah, I will leave removing this + the corresponding handlers to a later timestamp when we are closer to launch 😄